v3/mqtt_dummy.go

```diff
@@ -190,16 +191,26 @@ func handleSession(context *DummyContext, client *DummyClient) {
 
 		if err != nil {
 			logError("[MQTTD] Error reading packet from stream: %s", err)
-			keepConn = false
+			//keepConn = false
```

Comment:
Can we continue in this error case? Actually, I guess the client will cause this error, so I guess it's kind of a made up error?

Reply:
Sorry, I forgot to remove it, it' for debug. But `break` is available on next line, so eventually this loop is finished on this case. Isn't it?

Reply:
yep!